### DISCUSSION TITLE
[Bug]: QiankunError: application 'xxx' died in status LOADING_SOURCE_CODE: [qiankun]: You need to export lifecycle functions in xxx entry

### DISCUSSION THREAD

**Discussion by Ru1n33** [web-infra-dev/rsbuild#3919] [Q&A] [answered]
![1731028506333](https://github.com/user-attachments/assets/d52e8aa5-b8ea-4c68-99f2-43318055cdc8)
vue2项目接入qiankun，output这样配置的
![1731028578493](https://github.com/user-attachments/assets/5ec22c9a-5c09-44a8-9a67-5d3f9b410bf8)
主项目的名称一致，为什么在主项目里启动子项目一直会报这个错误，有大佬帮忙看一下嘛？

**Comment by Ru1n33** [comment, marked as answer]
解决了

**Comment by shiwenna** [comment]
遇到同样问题，请问一下怎么解决的？
